tests/Agent/IntegrationTests/UnboundedServices/example-secrets.json

```diff
@@ -0,0 +1,72 @@
+{
+    "IntegrationTestConfiguration": {
+      "DefaultSetting": {
+        "LicenseKey": "YOUR_LICENSE_KEY_HERE",
+        "Collector": "collector.newrelic.com",
+        "NewRelicAccountId" : "YOUR_ACCOUNT_ID_HERE"
+      },
+      "TestSettingOverrides": {
+        "PostgresTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "Server=127.0.0.1;Port=5432;User Id=postgres;Password=password;Database=postgres;"
+          }
+        },
+        "MSSQLTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "Server=127.0.0.1;Database=NewRelic;User ID=sa;Password=password0TS;Trusted_Connection=False;Encrypt=False;Connection Timeout=30;"
+          }
+        },
+        "MSSQLOdbcTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "DRIVER={SQL Server Native Client 11.0};Server=127.0.0.1;Database=NewRelic;Trusted_Connection=no;UID=sa;PWD=password0TS;Encrypt=no;"
+          }
+        },
+        "MSSQLOleDbTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "PROVIDER=SQLNCLI11;Server=127.0.0.1;Database=NewRelic;Trusted_Connection=no;UID=sa;PWD=password0TS;Encrypt=no;Timeout=30;"
+          }
+        },
+        "MySQLTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "Network Address=127.0.0.1;Port=3306;Initial Catalog=newrelic;Persist Security Info=no;User Name=root;Password=password;SslMode=None"
+          }
+        },
+        "OracleTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "Data Source=127.0.0.1:1521/XE;User Id=SYSTEM;Password=password;"
+          }
+        },
+        "StackExchangeRedisTests" : {
+          "CustomSettings": {
+              "ConnectionString" : "127.0.0.1:6379"
+          }
+        },
+        "Db2Tests" : {
+          "CustomSettings": {
+              "ConnectionString" : "Server=127.0.0.1;Database=SAMPLE;UserID=newrelic;Password=password"
+          }
+        },
+        "MongoDBTests" : {
```

Comment:
MongoDB is not needed since there are no tests for it and the containers were not added to the net35 branch

Reply:
Good catch, fixed.